ibis/backends/dask/execution/window.py

```diff
@@ -0,0 +1,86 @@
+"""Code for computing window functions in the dask backend."""
+
+from typing import Any, Optional, Union
+
+import dask.dataframe as dd
+
+import ibis.expr.operations as ops
+import ibis.expr.window as win
+from ibis.expr.scope import Scope
+from ibis.expr.typing import TimeContext
+
+from ..core import execute_with_scope
+from ..dispatch import execute_node
+from .util import (
+    _pandas_dtype_from_dd_scalar,
+    _wrap_dd_scalar,
+    add_partitioned_sorted_column,
+    make_meta_series,
+)
+
+
+def _post_process_empty(
+    result: Any,
+    parent: Union[dd.Series, dd.DataFrame],
+    timecontext: Optional[TimeContext],
+) -> dd.Series:
+    """Post process non grouped, non ordered windows.
+
+    dd.Series/dd.DataFrame objects are passed through, otherwise we conform
+    the output to the parent input (i.e. so the shape an partitioning matches).
+
+    dd.core.Scalar needs special handling so downstream functions can work
+    with it.
+    """
+    if isinstance(result, (dd.Series, dd.DataFrame)):
+        return result
+    elif isinstance(result, dd.core.Scalar):
```

Comment:
Can you explain why do we need to add the index column for any Scalar results?

Reply:
A quick google search of "Dask core Scalar" doesn't tell me much. Can you please explain this concept?

Reply:
e.g, is it just wrapping a scalar or can this also be a result of an aggregation?